.github/workflows/dashwallet.yml

```diff
@@ -0,0 +1,83 @@
+name: Dash Wallet CI
+
+on:
+  push:
```

Comment:
Though we don't need this file for the DashPay branch, perhaps we should leave it in the DashPay Branch.  Otherwise, when we have completed DashPay and merge to master, this file will be gone and CI will be broken.